lib/config-validator.ts

```diff
@@ -20,6 +20,7 @@ async function validate(
   desc: string,
   config: RenovateConfig,
   strict: boolean,
+  isGlobalConfig: boolean,
```

Comment:
```suggestion
  isGlobalConfig: boolean,
  desc: string,
  config: RenovateConfig,
  strict: boolean,
```

Please reorder like this, or refactor the code in a seprate PR so that the isPreset boolean is no longer optional. It's too easy to miss mistakes this way